.github/workflows/main.yml

```diff
@@ -0,0 +1,81 @@
+# This GitHub Actions workflow contains two jobs:
+# -- First, Lint uses the latest Ubuntu image and Python 3.9 to check
+# the code and writing for defects, using the Poetry lint task
+#
+# -- Second, Test uses a strategy matrix to test the code with different
+# Python versions and determine code coverage, using the Poetry test task
+
+name: Lint and Test
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Python 3.9
+        uses: actions/setup-python@v2
+        id: setup-python
+        with:
+          python-version: 3.9
```

Comment:
Is this the agreed on python version for the project or are you planning to use newer ones?

Reply:
This should be fine to check linting on though it could be changed to something more recent however I'm not sure it's completely important in this case.